proto/buf.yaml

```
@@ -0,0 +1,35 @@
+version: v1
+name: buf.build/osmosis-labs/sqs
+deps:
+  - buf.build/cometbft/cometbft:4a62c99d422068a5165429b62a7eb824df46cca9 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)